.github/CODEOWNERS

```diff
@@ -0,0 +1,9 @@
+# iOS
+/platform/ios/ @maps-ios-reviewers
```

Comment:
I believe these groups need to be qualified with the organization, e.g., `@mapbox/maps-ios-reviewers`.

/cc @mapbox/maps-ios-reviewers 😛